### ISSUE TITLE
Changing encrypted in aws_redshift_cluster triggers ForceNew

### ISSUE THREAD

**Issue by bjaworski3** [terraform-providers/terraform-provider-aws#6857]
Changing `encrypted` in `aws_redshift_cluster` from `false` to `true` triggers a ForceNew. Previously this was an unsupported change (See https://github.com/terraform-providers/terraform-provider-aws/issues/1119). The AWS docs now indicate that this is a supported change and I was able to do this manually to an existing cluster. The ForceNew flag should be removed from `encrypted`.

Reference Doc: https://docs.aws.amazon.com/redshift/latest/mgmt/working-with-db-encryption.html

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.11.10
+ provider.aws v1.52.0

### Affected Resource(s)

* aws_redshift_cluster

### Terraform Configuration Files

```hcl
resource "aws_redshift_cluster" "cluster" {
  cluster_identifier                  = "${var.name}"
  node_type                           = "${var.node_type}"
  database_name                       = "${var.database}"
  master_username                     = "${var.user}"
  master_password                     = "${var.pass}
  number_of_nodes                     = 1
  availability_zone                   = "${var.zone}"
  allow_version_upgrade               = true
  automated_snapshot_retention_period = 1
  cluster_public_key                  = "${var.public_key}"
  cluster_version                     = "1.0"
  cluster_type                        = "single-node"
  encrypted                           = false
  enhanced_vpc_routing                = false
  vpc_security_group_ids              = ["${aws_security_group.redshift.id}"]
  port                                = 5439
  preferred_maintenance_window        = "tue:07:00-tue:07:30"
  publicly_accessible                 = true
  skip_final_snapshot                 = true
  cluster_parameter_group_name        = "${var.parameter_group}"
  cluster_subnet_group_name           = "${var.subnet_group}"

  logging {
    enable        = true
    bucket_name   = "${var.bucket}"
    s3_key_prefix = "${var.s3_prefix}"
  }

  lifecycle {
    ignore_changes = [
      "cluster_public_key",
    ]
  }
}
```

### Expected Behavior

```
~ module.redshift.aws_redshift_cluster.cluster
    encrypted: "false" => "true"
```

### Actual Behavior

```
-/+ module.redshift.aws_redshift_cluster.cluster
    encrypted: "false" => "true"  (forces new resource)
```

### Steps to Reproduce

1. Create aws_redshift_cluster with encrypted = false
2. `terraform apply`
3. Change encrypted to true
4. `terraform plan`

**Comment by bflad** [comment]
Support for this has been merged in and will release with version 1.54.0 of the AWS provider, early January after the holiday break. 👍

**Comment by bflad** [comment]
Releasing early! This has been released in [version 1.54.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1540-december-21-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
